styleguide/_themes/global/scss/bootstrap_variables.scss

```diff
@@ -887,9 +887,3 @@ $hr-border:                   $gray-lighter !default;
 
 
 $font_path: 'https://043a96a8a2caa9c57463-9701ce6e6cbd7003c5a8a25a5a69546c.ssl.cf5.rackcdn.com/v1.1/fonts/';
-
-$brand-black: $gray-base;
-$brand-yellow: #f6b100;
-$brand-yellow-dark: #f6a100;
-$brand-blue: #1ae9ef;
-$brand-blue-dark: #0dcfd9;
```

Comment:
@WidgetsBurritos is correct. This PR should only contain changes necessary for the task they are associated with. I would go ahead and create a JIRA task to strip out unused sass variables.